app/src/components/card/searchCard.test.tsx

```diff
@@ -131,38 +131,36 @@ describe("Search card displaying highlighted text for keywords", () => {
   });
 
   it("displays text with highlight field in card", () => {
-    render(
+    const component = (
```

Comment:
nit: follow the convention from other tests where it's `render(<....`, so no need to create the variable.